clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/validators/StandardAppEngineAttributeValidator.groovy

```diff
@@ -233,4 +236,32 @@ class StandardAppEngineAttributeValidator {
     }
     return valid
   }
+
+  def validateServingStatusCanBeChanged(String serverGroupName,
+                                        AppEngineNamedAccountCredentials credentials,
+                                        AppEngineClusterProvider clusterProvider,
+                                        String attribute) {
+    def valid = true
```

Comment:
You could also try to refactor this function to use a single `return valid` statement, but that might get messy in this case.